core/src/main/java/me/leoko/advancedban/ServerType.java

```diff
@@ -0,0 +1,20 @@
+package me.leoko.advancedban;
+
+public enum ServerType {
+
+  BUNGEECORD("Bungeecord"),
+  VELOCITY("Velocity"),
+  SPIGOT("Spigot"),
+  TEST("Test");
+
+  private String serverType;
+
+  private ServerType(String serverType) {
+    this.serverType = serverType;
+  }
+
+  public String getString() {
```

Comment:
toString() would make more sense here than getString

Reply:
Agreed, will change this on the next commit